mica-webapp/src/main/resources/i18n/en.json

```diff
@@ -1079,6 +1079,14 @@
         "title": "Signature",
         "check": "I have completed all mandatory fields, included all necessary documents, and hereby submit this application."
       }
+    },
+    "required-documents": {
```

Comment:
move to "data-access-request.default" which group the default DAR form translations.